java/google/registry/model/eppoutput/Greeting.java

```diff
@@ -42,8 +45,9 @@
   @XmlElement
   static Dcp dcp = new Dcp();
 
-  public static Greeting create(DateTime svDate) {
+  public static Greeting create(DateTime svDate, @Config("greetingServerId") String svID) {
```

Comment:
Hold on, are you sure that this works?  This is neither field nor constructor injection, so I don't think it's actually doing anything.